src/main/resources/schema/elytron-client-1_3.xsd

```diff
@@ -553,7 +553,7 @@
                 </xsd:documentation>
             </xsd:annotation>
         </xsd:attribute>
-        <xsd:attribute name="type" type="xsd:string" use="required">
+        <xsd:attribute name="type" type="xsd:string" use="optional">
             <xsd:annotation>
                 <xsd:documentation>
                     The key store type.
```

Comment:
I think it is worth describe little bit, that it is optional only for some of file-based keystores. For non-filebased keystores it is still required.